pkg/sloop/common/partition_utilities.go

```diff
@@ -0,0 +1,87 @@
+package common
+
+import (
+	"github.com/dgraph-io/badger/v2"
+	"github.com/golang/glog"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
+	"sort"
+)
+
+type SloopKey struct {
+	TableName   string
+	PartitionID string
+}
+
+// returns TableName, PartitionId, error.
+func GetSloopKey(item badgerwrap.Item) (SloopKey, error) {
+	key := item.Key()
+	err, parts := ParseKey(string(key))
+	if err != nil {
+		return SloopKey{}, err
+	}
+
+	var tableName = parts[1]
+	var partitionId = parts[2]
+	return SloopKey{tableName, partitionId}, nil
+}
+
+type PartitionInfo struct {
+	TotalKeyCount          uint64
+	TableNameToKeyCountMap map[string]uint64
+}
+
+// prints all the keys histogram. It can help debugging when needed.
+func PrintKeyHistogram(db badgerwrap.DB) {
+	partitionTableNameToKeyCountMap, totalKeyCount := GetPartitionsInfo(db)
+	glog.Infof("TotalkeyCount: %v", totalKeyCount)
+
+	for partitionID, partitionInfo := range partitionTableNameToKeyCountMap {
+		for tableName, keyCount := range partitionInfo.TableNameToKeyCountMap {
+			glog.Infof("TableName: %v, PartitionId: %v, keyCount: %v", tableName, partitionID, keyCount)
```

Comment:
This will print out a lot of keys by default, maybe we can add glog.V(some#)so we only print when needed.